### PR TITLE
Twitter

### PR DESCRIPTION
**Pull request by chriskonstad**
This is the same PR that @bmwwoo made.  I merged it, but it caused some bugs missed in testing so I undid the merge and am now re-opening the PR for conversation/fixing.  Sorry for the rebasing and force pushing to get things back to this state :fearful:

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
+
   devise_for :users
-  get 'market_chirp/index'
+  get 'market_chirp/index', as: :market_chirp
```
Comment (chriskonstad) [inline config/routes.rb]: This changes the URL, and our market_chirp controller is our root controller.  We probably shouldn't do this.

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
-  get 'market_chirp/index'
```
Comment (chriskonstad) [inline config/routes.rb]: We don't need this as setting the root to this route automatically handles get for us, plus we don't want to get 'market_chirp/index'.

**Comment by bmwwoo** [comment]
LGTM :v:

**Comment by chriskonstad** [comment]
okay, you want to merge it?

**Pull request merged**